core/math/math_funcs.h

```diff
@@ -390,20 +390,9 @@ class Math {
 
 #if (defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x0603) || WINAPI_FAMILY == WINAPI_FAMILY_PHONE_APP // windows 8 phone?
 		b = (int)((a > 0.0) ? (a + 0.5) : (a - 0.5));
```

Comment:
This effectively means we always use the manual path for Windows > `0x0603`, i.e. Windows 10. I'm not sure why?

`lrintf` should be supported for both C and C++ in all supported MSVC versions: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/lrint-lrintf-lrintl-llrint-llrintf-llrintl?view=msvc-160

Reply:
This seems to be a bogus check added while attempting to port to WinRT: 089d7fa171e3a3305991047e82b4043d4f05783f.

Reply:
It seems `lrintf` support was added to WinRT in the first release of Windows 10: https://github.com/MicrosoftDocs/winrt-related/blob/docs/winrt-related-src/win32-and-com/win32-extension-apis.md

So the fallback would still be needed for Windows 8 WinRT, but maybe we don't care? In any case it shouldn't apply to all Windows platforms like now.

Reply:
> So the fallback would still be needed for Windows 8 WinRT, but maybe we don't care? In any case it shouldn't apply to all Windows platforms like now.

Windows 8 RT is EOL and I doubt anyone is developing Godot projects for it these days.